draft-ietf-quic-multipath.md

```diff
@@ -353,13 +353,16 @@ as "no packet received on any path for the duration of the idle timeout".
 When only one path is available, servers MUST follow the specifications
 in {{QUIC-TRANSPORT}}.
 
-When more than one path is available, servers shall monitor the arrival
-of non-probing packets on the available paths. Servers SHOULD stop
-sending traffic on paths through where no non-probing packet was received
-in the last 3 path RTTs, but MAY ignore that rule if it would disqualify
+When more than one path is available, hosts shall monitor the arrival
+of non-probing packets  and the acknowledgements 
+for the packets sent over each path. Hosts SHOULD stop
+sending traffic on paths if (a) no non-probing packet was received
+in the last 3 path RTTs or (b) no non-probing packet sent over this
+path was acknowledged during the last 3 path 
+RTT, but MAY ignore that rule if it would disqualify
```

Comment:
```suggestion
When more than one path is available, hosts shall monitor the arrival
of non-probing packets and the acknowledgements 
for the packets sent over each path. Hosts SHOULD stop
sending traffic on a path if for at least max_idle_timeout milliseconds 
(a) no non-probing packet was received or (b) no non-probing
packet sent over this path was acknowledged, but MAY ignore that
rule if it would disqualify
```